docs/source/command_line.rst

```diff
@@ -21,7 +21,7 @@ flag (or its long form ``--help``)::
               [--warn-redundant-casts] [--no-warn-no-return] [--warn-return-any]
               [--warn-unused-ignores] [--warn-unused-configs]
               [--show-error-context] [--no-implicit-optional] [-i]
-              [--quick-and-dirty] [--cache-dir DIR] [--cache-fine-grained]
+              [--quick-and-dirty] [--cache-dir DIR]
```

Comment:
Um, `--cache-fine-grained` is not really undocumented (it has a help message), so you can put it back.

Reply:
It was never added as a flag to these docs so I wasn't sure if I should add it here, but I will do so!